sdk/resourcemanager/Azure.ResourceManager.Core/src/ResourceListOperations.cs

```diff
@@ -144,60 +137,22 @@ private static ResourcesManagementClient GetResourcesClient(ResourceOperationsBa
             int? top = null,
             CancellationToken cancellationToken = default)
         {
-            var armOperations = GetResourcesClient(resourceOperations).Resources;
-            Pageable<GenericResourceExpanded> result;
+            var restClient = GetRestClient(resourceOperations);
```

Comment:
nit: RestClient throw me off a littlebit. Can we name it GetGenericResourceContainer?